### PR TITLE
Update EL6 installation instructions for new Foreman location

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@jlsherrill @johnpmitsch if you care to peruse

File: docs/installation/index.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ yum-config-manager --enable rhel-6-server-optional-rpms
 <div id="el6" markdown="1">
 ```bash
 yum -y localinstall http://fedorapeople.org/groups/katello/releases/yum/{{ site.version }}/katello/el6/x86_64/katello-repos-latest.rpm
-yum -y localinstall http://yum.theforeman.org/releases/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
+yum -y localinstall http://fedorapeople.org/groups/katello/releases/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
```
Comment (johnpmitsch) [inline docs/installation/index.md]: I don't see a formean directory here? https://fedorapeople.org/groups/katello/releases/ 

I think you mean https://fedorapeople.org/groups/katello/releases/yum/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm ?

File: docs/upgrade/capsule.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Update the Foreman and Katello release packages:
 
 ```
   # yum update -y http://fedorapeople.org/groups/katello/releases/yum/{{ site.version }}/katello/el6/x86_64/katello-repos-latest.rpm
-  # yum update -y http://yum.theforeman.org/releases/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
+  # yum update -y http://fedorapeople.org/releases/yum/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
```
Comment (johnpmitsch) [inline docs/upgrade/capsule.md]: I think you mean

https://fedorapeople.org/groups/katello/releases/yum/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm

File: docs/upgrade/index.md [hunk 3]
```diff
@@ -37,12 +37,12 @@ Ensure your operating system is fully up-to-date:
...
 Update the Foreman and Katello release packages:
 
-  * RHEL6 / CentOS 6 **Foreman packages are not available yet**:
+  * RHEL6 / CentOS 6:
 
 ```
 
   # yum update -y http://fedorapeople.org/groups/katello/releases/yum/{{ site.version }}/katello/el6/x86_64/katello-repos-latest.rpm
-  # yum update -y http://yum.theforeman.org/releases/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
+  # yum update -y http://fedorapeople.org/releases/yum/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
```
Comment (johnpmitsch) [inline docs/upgrade/index.md]: same as above - do  you mean

https://fedorapeople.org/groups/katello/releases/yum/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm ?

**Comment by ehelms** [comment]
@johnpmitsch updated

**Comment by johnpmitsch** [comment]
ACK

**Pull request merged**